testdata/picard/sam/namesorted.test.sam

```diff
@@ -25,9 +25,9 @@
-readpair2	99	1	1811000	29	58M43S3H	=	1811263	118	CGTGGTGGCAGGCACCTGTAGTCCCAGCTATTCGGGAGGCTGAGGCAGGAGAATGGCGTGAACCCAGGAGGCGGACCTTGCAGTGAGCCAAGATCGACCCA	AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA############################################	X0:i:4	X1:i:200	XC:i:58	MD:Z:58	RG:Z:20FUK.1	XG:i:0	AM:i:0	NM:i:0	SM:i:0	XM:i:0	XO:i:0	BQ:Z:@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
-readpair2	147	1	1811030	37	4H12S89M	=	1811004	-118	GATCGTCACATCGTGTCTGCTCTCAGCCTGATGCAATACACAGTTTTAGCTGAAGTCTAAGAGGAAAATTCAGTCTCACATAGATATGCAGCTGGAAAAAG	#############AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA	X0:i:1	X1:i:0	XC:i:89	MD:Z:89	RG:Z:20FUK.1	XG:i:0	AM:i:0	NM:i:0	SM:i:37	XM:i:0	XO:i:0	BQ:Z:@@@@@@@@@@@@>YV[Z`X[aXQWXRBGUTX^V]^^VNXRXXSVVUHWQWVUZZX]WHVXUXRUUUV[VJ\XRPXZXTQWWWTVTT_X^_]XOXWWZZY[Z
-readpair2Dup	1123	1	1811000	29	58M43S3H	=	1811263	118	CGTGGTGGCAGGCACCTGTAGTCCCAGCTATTCGGGAGGCTGAGGCAGGAGAATGGCGTGAACCCAGGAGGCGGACCTTGCAGTGAGCCAAGATCGACCCA	AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA############################################	X0:i:4	X1:i:200	XC:i:58	MD:Z:58	RG:Z:20FUK.1	XG:i:0	AM:i:0	NM:i:0	SM:i:0	XM:i:0	XO:i:0	BQ:Z:@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
-readpair2Dup	1171	1	1811030	37	4H12S89M	=	1811004	-118	GATCGTCACATCGTGTCTGCTCTCAGCCTGATGCAATACACAGTTTTAGCTGAAGTCTAAGAGGAAAATTCAGTCTCACATAGATATGCAGCTGGAAAAAG	#############AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA	X0:i:1	X1:i:0	XC:i:89	MD:Z:89	RG:Z:20FUK.1	XG:i:0	AM:i:0	NM:i:0	SM:i:37	XM:i:0	XO:i:0	BQ:Z:@@@@@@@@@@@@>YV[Z`X[aXQWXRBGUTX^V]^^VNXRXXSVVUHWQWVUZZX]WHVXUXRUUUV[VJ\XRPXZXTQWWWTVTT_X^_]XOXWWZZY[Z
+readpair1	99	1	1811000	29	58M43S	=	1811263	118	CGTGGTGGCAGGCACCTGTAGTCCCAGCTATTCGGGAGGCTGAGGCAGGAGAATGGCGTGAACCCAGGAGGCGGACCTTGCAGTGAGCCAAGATCGACCCA	AAAAAAAAAAAAAAAAAAAAAAAAAAAAA0000000000AAAAAAAAAAAAAAAAAA############################################	X0:i:4	X1:i:200	XC:i:58	MD:Z:58	RG:Z:20FUK.1	XG:i:0	AM:i:0	NM:i:0	SM:i:0	XM:i:0	XO:i:0	BQ:Z:@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
+readpair1	147	1	1811030	37	12S89M	=	1811004	-118	GATCGTCACATCGTGTCTGCTCTCAGCCTGATGCAATACACAGTTTTAGCTGAAGTCTAAGAGGAAAATTCAGTCTCACATAGATATGCAGCTGGAAAAAG	#############AAAAAAAAAAAA0000000000AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA	X0:i:1	X1:i:0	XC:i:89	MD:Z:89	RG:Z:20FUK.1	XG:i:0	AM:i:0	NM:i:0	SM:i:37	XM:i:0	XO:i:0	BQ:Z:@@@@@@@@@@@@>YV[Z`X[aXQWXRBGUTX^V]^^VNXRXXSVVUHWQWVUZZX]WHVXUXRUUUV[VJ\XRPXZXTQWWWTVTT_X^_]XOXWWZZY[Z
```

Comment:
Sorry to be picky. Could you please add some unmapped reads, secondary reads, and vendor-failed reads?